### DISCUSSION TITLE
part-ii/2021-104/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#379] [Question comments]
Part II 2021 Number Theory: Paper 2, Section $I$, I 

https://questions.tripos.org/part-ii/2021-104/

**Comment by Kiolt** [comment]
<details>
<summary>Identity</summary>

Show that the function is multiplicative and check prime powers.

</details>

<details>
<summary>Rider</summary>

CRT so that different prime squares divide $n,n+1,\ldots,n+k$.

</details>
